README.md

```diff
@@ -223,6 +223,41 @@ Mount the created secret to your Fluentd container
 oc volumes dc/logging-fluentd --add --type=secret --secret-name=fluentd-throttle --mount-path=/etc/throttle-settings --name=throttle-settings --overwrite
 ```
 
+## Have Fluentd send logs to another Elasticsearch
+
+You can configure Fluentd to send a copy of each log message to another
+Elasticsearch instance, _not_ the instance included with the log aggregation
+deployment.  Perhaps you already have an Elasticsearch instance that you are
```

Comment:
I don't think this is the correct use case for this feature.  To me, this is for when there are two different Elasticsearch instances that should be receiving the same logs, such as for auditing.  To simply send logs from the OpenShift fluentd to an external ES, we could just modify the current env vars.

I would make mention of sending copies of the logs to both the OpenShift hosted and an external.